Source/WebCore/Modules/compression/CompressionStreamEncoder.cpp

```diff
@@ -164,4 +176,82 @@ ExceptionOr<Ref<JSC::ArrayBuffer>> CompressionStreamEncoder::compress(std::span<
 
     return compressedData.releaseNonNull();
 }
+
+
+#if PLATFORM(COCOA)
+ExceptionOr<bool> CompressionStreamEncoder::initializeAppleCompressionFramework() 
```

Comment:
This kind of thing should probably go in a platform file like CompressionStreamEncoderCocoa.cpp